tests/robustness/failpoint/cluster.go

```diff
@@ -228,13 +229,32 @@ func (f memberDowngradeUpgrade) Inject(ctx context.Context, t *testing.T, lg *za
 
 	e2e.DowngradeEnable(t, clus, lastVersion)
 	// downgrade all members first
-	err = e2e.DowngradeUpgradeMembers(t, lg, clus, len(clus.Procs), currentVersion, lastVersion)
+	err = e2e.DowngradeUpgradeMembers(t, lg, clus, len(clus.Procs), e2e.BinPath.EtcdLastRelease, version.Versions{
+		Cluster: lastVersion.String(),
+		Server:  lastVersion.String(),
+	})
 	if err != nil {
 		return nil, err
 	}
+
+	// NOTE: By default, the leader can cancel the downgrade once all members
+	// have reached the target version. However, determining the final stable
+	// cluster version after an upgrade can be challenging. To ensure stability,
+	// we should wait for leader to cancel downgrade process.
+	e2e.AssertProcessLogs(t, e2e.GetLeader(t, clus), "the cluster has been downgraded")
```

Comment:
can we check the downgrade status as well?

Reply:
v3.5 doesn't have this API change so it's unlikely to check status here.

Reply:
OK, got it.